internal/pipe/nfpm/nfpm.go

```diff
@@ -328,6 +341,14 @@ func create(ctx *context.Context, fpm config.NFPM, format, arch string, binaries
 	return nil
 }
 
+func sources(contents files.Contents) []string {
```

Comment:
only used in test code, I would move to the test file...

Reply:
Moved it to the test file, thanks for pointing that out.